src/pacer/api_integration.py

```
@@ -0,0 +1,398 @@
+                    "error": "PACER monitor not initialized"
+                }), 503
+            
+            stats = pacer_monitor.get_statistics()
+            return jsonify({
+                "success": True,
+                "data": {
+                    "monitor_status": "running" if stats["is_running"] else "stopped",
+                    "statistics": stats
+                }
+            })
+        except Exception as e:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)